### PR TITLE
New article for habitica

### PR DESCRIPTION
**Pull request by AdrienLemaire**
[Preview here](http://adrien.is/2015-09-31-habitica-habit-building-program.html)

### CODE REVIEW

File: 2015-09-31-habitica-habit-building-program.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+---
+
+`Well... I'm already using Habitica, so...`<br />
+Wait! I've got a gift for you :smile: Go directly to [the tips](#tips).
+
+# The minimalist guide to Habitica
+
+[Habitica](https://habitica.com) is a todo-list
+application inspired from the [Get Things Done](http://gettingthingsdone.com/)
+productivity methodology.
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: so what's habitRPG from the title?

File: 2015-09-31-habitica-habit-building-program.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+* **Dailies**: for recurring activities.
+* **Todo**: for one-time tasks.
+* **Rewards**: for leisure activities available after working hard on your tasks.
+
+It's  simple to use, accepts the [Markdown](http://daringfireball.net/projects/markdown/) 
+markup language and [Emoji](http://emoji-cheat-sheet.com/) emoticons.
+
+
+Although its todo-list features have increased a lot over the time, I believe the
+strenght of Habitica comes from the game that wraps the tool, and the community
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: strength

comes from the gaming aspect

File: 2015-09-31-habitica-habit-building-program.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+As a fact, I've tried many todo-list tools before Habitica, and always gave
+up on them (missing features, not intuitive enough, or simply because I couldn't
+build the habit of checking them regularly and forgot them). But when I started
+using Habitica back in its beta days, although it was cribbled with bugs, I
+sticked to it because I was having fun and had met very friendly people there.
+
+{: .clear}
+![Habitica Logo](/images/habitica/logo.png "New logo looking like a Gryphon"){: .float_left}
+Formely named **HabitRPG**, the application was renamed **Habitica** on July 31th 2015.
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: should be in the intro

File: 2015-09-31-habitica-habit-building-program.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+multiples aspects:
+
+* **Health**, **Experience**, **Mana**: Like any [RPG
+  game](https://en.wikipedia.org/wiki/Role-playing_game), your avatar will
+  evolve as you complete tasks, and will die if you're slacking off.
+
+  {: .centered}
+  ![Health, XP and Mana bars](/images/habitica/health_xp_mana.png)
+* **Attribute points and Class System**: Do you want to complete quests faster,
+  get more gold, avoid dying? Smartly distribute your points in [Strenght,
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: strength

File: 2015-09-31-habitica-habit-building-program.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+  Intelligence, Constitution and Perception](http://habitica.wikia.com/wiki/Character_Attributes),
+  and select the most appropriate class for your avatar
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: so what do these classes do ?

File: 2015-09-31-habitica-habit-building-program.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+  feed pets into mounts, buy equipments, and enjoy customizing your avatar in
+  any way you'd like.
+* **Party**, **Guilds**: You can join groups of people from all over the world,
+  doing completely different jobs, and socialize with them, motivate each other
+  to become everyday a bit more productive.
+
+  {: .centered}
+  ![Party backgrounds](/images/habitica/party.png)
+* **Quests**, **Challenges**: Fight against bosses by completing your tasks,
+  together with your team members, or compete against other players for habit
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: too many commas

File: 2015-09-31-habitica-habit-building-program.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+
+  {: .centered }
+  ![Classes system: Warrior, Mage, Rogue and Healer](/images/habitica/classes.png)
+* **Gold**, **Food**, **Eggs**, **Potions**: As you complete your tasks, items
+  will drop.
+ 
+  {: .centered }
+  ![items](/images/habitica/items.png)
+* **Pets**, **Mounts**, **Equipment**, **Background**: Hatch eggs into pets,
+  feed pets into mounts, buy equipments, and enjoy customizing your avatar in
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: buy equipments and enjoy (no comma)

File: 2015-09-31-habitica-habit-building-program.md [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+  any way you'd like.
+* **Party**, **Guilds**: You can join groups of people from all over the world,
+  doing completely different jobs, and socialize with them, motivate each other
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: different jobs, socialize with them and motivate each other

File: 2015-09-31-habitica-habit-building-program.md [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+  building activities.
+* **Achievements badges**: You can collect badges by contributing, by killing
+  bosses, by getting all pets/mounts/equipments, by interacting with your party,
+  and more!
+
+  {: .centered}
+  ![Achievement badges](/images/habitica/badges.png)
+
+
+# An excerpt of my Personal experience
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: no capital P

File: 2015-09-31-habitica-habit-building-program.md [hunk 10]
```diff
@@ -0,0 +1,204 @@
...
+
+What hooked me up the most are the facts that Habitica is Open Source and
+strongly motivates you to contribute, as well as the community.
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: What hooked me up the most was the community and that Habitica is open source and strongly encourages you to contribute.

File: 2015-09-31-habitica-habit-building-program.md [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+
+{: .clear}
+![Project Mayhem Logo](/images/habitica/pm.png "Using the quest boss Vice"){: .float_left}
+I had the chance to join a really great party named
+[Project Mayhem](https://hrpgprojectmayhem.wordpress.com/), and we went through
+countless quests, looking for holes in the game to break it, and always keeping
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: don't put comma before a 'and'

File: 2015-09-31-habitica-habit-building-program.md [hunk 12]
```diff
@@ -0,0 +1,204 @@
...
+
+The beauty about [contributions](http://habitica.wikia.com/wiki/Contributing_to_HabitRPG)
+is that anybody can participate: Designers, Pixel artists, Translators, Music
+and soud effect artists, Writers, Mobile & Web coders. Since the game has been
+documented by its users, it results in
+[one of the best documentations](http://habitica.wikia.com/wiki/Habitica_Wiki)
+I've ever seen for an application.
+
+Once I've contributed to the project, of course it made me want to use it more,
+and I've borne the eventual bugs, finding new ways to handle my tasks the way I
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: borne?

File: 2015-09-31-habitica-habit-building-program.md [hunk 13]
```diff
@@ -0,0 +1,204 @@
...
+wanted.
+
+# The behind-the-scenes of Habitica
+
+The code source for the web version is built on top of
+[AngularJS](https://angularjs.org/). Mobile versions were initially made by
+contributors, then [an official one](https://github.com/HabitRPG/habitrpg-mobile)
+came out from AngularJS and [PhoneGap](http://phonegap.com/). Up to today,
+I have to say this application has given me lot of troubles, especially its
+slowliness (the flip side of using multi-platform frameworks). But there are
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: slowness
the downside of using a HTML app

File: 2015-09-31-habitica-habit-building-program.md [hunk 14]
```diff
@@ -0,0 +1,204 @@
...
+good news: the native [iOS application](https://itunes.apple.com/us/app/habitica/id994882113?mt=89)
+is now available, and the android one is on its way (last announce on July 22th).
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: last announce? what do you want to say ?

File: 2015-09-31-habitica-habit-building-program.md [hunk 15]
```diff
@@ -0,0 +1,204 @@
...
+
+I'll definitely have a look at the Android app and do some minor contributions
+when I'll find time, this will be a good use-case for the [UX Design for Mobile
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: when I'll find time as this will be

File: 2015-09-31-habitica-habit-building-program.md [hunk 16]
```diff
@@ -0,0 +1,204 @@
...
+*Go to your pets page, open your browser debugger (ctrl-shift-I on Chrome), go to
+the Console, paste the previous code there and press enter. Click on a
+food, then click on all pets you'd like to feed with that food until it runs
+out.*
+
+
+
+Did you find this article interesting ? If you haven't heard of Habitica before,
+does it make you want to give it a try ? Otherwise if you're already a user, did
+you find the tips useful ?
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: too many questions

File: 2015-09-31-habitica-habit-building-program.md [hunk 17]
```diff
@@ -0,0 +1,204 @@
...
+I will blog regularly about Productivity applications and methodologies, so if
+you'd like to stay tuned, [please subscribe here](http://eepurl.com/bt8739).
```
Comment (Keats) [inline 2015-09-31-habitica-habit-building-program.md]: eepurl.com? what's that

File: 2015-09-31-habitica-habit-building-program.md (hunk 12 above)
Comment (AdrienLemaire) [inline 2015-09-31-habitica-habit-building-program.md]: conjugation of the verb "to bear" ^^

File: 2015-09-31-habitica-habit-building-program.md (hunk 17 above)
Comment (AdrienLemaire) [inline 2015-09-31-habitica-habit-building-program.md]: mailchimp :)

File: 2015-09-31-habitica-habit-building-program.md (hunk 2 above)
Comment (AdrienLemaire) [inline 2015-09-31-habitica-habit-building-program.md]: nice catch! Thanks :)

File: 2015-09-31-habitica-habit-building-program.md (hunk 5 above)
Comment (AdrienLemaire) [inline 2015-09-31-habitica-habit-building-program.md]: Not sure if I should develop this part. But I'll add a link to the tips section, and add more advanced tips on what class to use in which circumstance, thanks!.

File: 2015-09-31-habitica-habit-building-program.md (hunk 7 above)
Comment (AdrienLemaire) [inline 2015-09-31-habitica-habit-building-program.md]: ran a `s/, and/ and/g`, thanks ^^

File: 2015-09-31-habitica-habit-building-program.md (hunk 8 above)
Comment (AdrienLemaire) [inline 2015-09-31-habitica-habit-building-program.md]: thx

File: 2015-09-31-habitica-habit-building-program.md (hunk 14 above)
Comment (AdrienLemaire) [inline 2015-09-31-habitica-habit-building-program.md]: :+1: reworded

File: 2015-09-31-habitica-habit-building-program.md (hunk 1 above)
Comment (AdrienLemaire) [inline 2015-09-31-habitica-habit-building-program.md]: fixed, thanks :)

**Comment by ghost** [comment]
The title was a bit confusing to me, I thought HabitRPG is shutting down and building something else. And the second title I'm not sure if "minimalist guide" makes sense.

**Pull request merged**